tests/large/test_ide.py

```diff
@@ -31,26 +31,25 @@
 logger = logging.getLogger(__name__)
 
 
-class EclipseIDETests(LargeFrameworkTests):
-    """The Eclipse distribution from the IDE collection."""
-
+class BaseLargeIdeTest(LargeFrameworkTests):
     TIMEOUT_INSTALL_PROGRESS = 120
     TIMEOUT_START = 60
     TIMEOUT_STOP = 60
 
     def setUp(self):
         super().setUp()
-        self.installed_path = os.path.expanduser("~/tools/ide/eclipse")
-        self.desktop_filename = "eclipse.desktop"
+        self.name = ""
+        self.command_name = ""
+        self.installed_path = ""
+        self.desktop_filename = ""
 
     @property
     def arch_option(self):
         """we return the expected arch call on command line"""
         return platform.machine()
 
-    def test_default_eclipse_ide_install(self):
-        """Install eclipse from scratch test case"""
-        self.child = pexpect.spawnu(self.command('{} ide eclipse'.format(UMAKE)))
+    def test_default_install(self):
+        self.child = pexpect.spawnu(self.command('{0} ide {1}'.format(UMAKE, self.name)))
```

Comment:
Please keep the coherence with the rest of the umake style and don't create any uncessary diff: when we only have few arguments to a string, only use {}, no {index}.